### ISSUE TITLE
demo_pt.py failed on ICNet

### ISSUE THREAD

**Issue by platvlad** [osmr/imgclsmob#87]
Hi!
I am trying to run demo script `examples/demo.py` on ICNet model `icnet_resnetd50b_cityscapes` for PyTorch:

<pre><code>cd examples
python demo_pt.py --model icnet_resnetd50b_cityscapes --image test.jpg --num-gpus 1
</code></pre>

I am getting an error:
<pre><code>File "/home/oem/repo/libs/ML/venv/lib/python3.8/site-packages/torch/nn/functional.py", line 3151, in interpolate
    return torch._C._nn.upsample_bilinear2d(input, output_size, align_corners, scale_factors)
TypeError: upsample_bilinear2d(): argument 'output_size' must be tuple of ints, but found element of type float at pos 1
</code></pre>
As I see in `InterpolationBlock` [code](https://github.com/osmr/imgclsmob/blob/7ddb5623aadc6a190715c0387dcc8776017115f9/pytorch/pytorchcv/models/common.py#L1399), `calc_out_size` returns a tuple of floats, since its elements are multiplied by float `self.scale_factor`, which is `0.5` for [some ICNet blocks](https://github.com/osmr/imgclsmob/blob/7ddb5623aadc6a190715c0387dcc8776017115f9/pytorch/pytorchcv/models/icnet.py#L233).
This cause tuple of floats to be provided to be provided into torch `interpolate` function [here](https://github.com/osmr/imgclsmob/blob/7ddb5623aadc6a190715c0387dcc8776017115f9/pytorch/pytorchcv/models/common.py#L1383).
My PyTorch version is 1.7.1.

Is it a bug of ICNet for Pytorch implementation?

**Comment by osmr** [comment]
Nope.

**Comment by osmr** [comment]
You try to use a script, designed to showcase the work of classification networks with standard ImageNet-1K pipeline, on a segmentation model.

**Comment by platvlad** [comment]
@osmr Thank you for your reply. 

I am modifying the script by deleting everything after running model on image data (starting from [this line](https://github.com/osmr/imgclsmob/blob/7ddb5623aadc6a190715c0387dcc8776017115f9/examples/demo_pt.py#L121)).

So I just load, crop and normalize the image and run model on it. I`m not trying to parse the model results as classification output, but still have this error.

In other words, I also have this error with the following code:

<pre><code>
import torch
from pytorchcv.model_provider import get_model as ptcv_get_model

net = ptcv_get_model('icnet_resnetd50b_cityscapes', pretrained=True).eval().cuda()
x = torch.rand((1, 3, 224, 224), dtype=torch.float).cuda()
net(x)
</code></pre>

**Comment by platvlad** [comment]
@osmr Thank you!
I have another error now, for which I opened a new issue #88.
